src/main/java/frc/robot/commands/Arm/StopNoteContainerMotor.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.Arm;
+import frc.robot.subsystems.ArmSubsystem;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class StopNoteContainerMotor extends Command {
+
+  
+
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final ArmSubsystem armSubsystem;
+
+  public StopNoteContainerMotor(ArmSubsystem armSubsystem) {
+    this.armSubsystem = armSubsystem;
+    addRequirements(armSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
```

Comment:
You can omit empty methods like this - there is an empty default in the super-class